website/modules/snackPlayerInitializer.js

```diff
@@ -65,8 +65,38 @@ export default (() => {
     });
   };
 
+  const setupSnackObserver = () => {
+    const observer = new MutationObserver(mutations => {
+      let shouldInitialize = false;
+
+      mutations.forEach(mutation => {
+        mutation.addedNodes.forEach(node => {
+          if (node.nodeType === 1) {
+            if (
+              node.classList?.contains('snack-player') ||
+              node.querySelector?.('.snack-player')
+            ) {
+              shouldInitialize = true;
+            }
+          }
+        });
+      });
+
+      if (shouldInitialize && window.ExpoSnack) {
+        updateSnacksTheme();
+        window.ExpoSnack.initialize();
+      }
+    });
+
+    observer.observe(document.body, {
```

Comment:
We should be able to downscope the observer pool, and just look inside doc content container, marked with `.container` class.